docs/docs/environment_config.md

```diff
@@ -24,6 +25,9 @@ Sceptre only checks for and uses the above keys in environment config files, but
 
 The ARN of a role for Sceptre to assume before interacting with the environment. If not supplied, Sceptre uses the user's AWS CLI credentials.
 
+### iam_profile
+
+The name of the local role as defined in ~/.aws/config and ~/.aws/credentials. If `iam_role` is also provided, Sceptre will use `iam_profile` to assume the `iam_role`. If `iam_role` is not provided, Sceptre will use `iam_profile` to interact with the environment.
```

Comment:
Few suggestions on wording:
- The name of the *profile*..... instead of *local role*
- Sceptre will use the `profile` to interact with AWS... instead of Sceptre will use `iam_profile` to interact with the environment.